src/databricks/labs/ucx/install.py

```diff
@@ -245,6 +245,15 @@ def _prompt_for_new_installation(self) -> WorkspaceConfig:
         configure_groups = ConfigureGroups(self.prompts)
         configure_groups.run()
         include_databases = self._select_databases()
+
+        # Checking if the user wants to define a default owner group.
+        default_owner_group = None
+        if self.prompts.confirm(
+            "Do you want to define a default owner group for all tables and schemas? "
```

Comment:
I have some doubts about the length of the question. How does this look in a terminal? It is good it is verbose, but it might be the only multi-line question we have depending on the size of your cli of course